roles/ci_dcn_site/defaults/main.yml

```diff
@@ -14,9 +14,9 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-ci_dcn_site_arch_repo_path: /home/zuul/src/github.com/openstack-k8s-operators/architecture
+ci_dcn_site_arch_repo_path: "{{ cifmw_architecture_repo | default('/home/zuul/src/github.com/openstack-k8s-operators/architecture') }}"
 ci_dcn_site_arch_path: "{{ ci_dcn_site_arch_repo_path }}/examples/dt/dcn"
-ci_dcn_site_cifmw_repo_path: /home/zuul/src/github.com/openstack-k8s-operators/ci-framework
+ci_dcn_site_cifmw_repo_path: "{{ ci_dcn_site_arch_repo_path | regex_replace('architecture$', 'ci-framework') }}"
```

Comment:
Finding issue here after a while might be difficult.
Could we leave `/home/zuul/src/github.com/openstack-k8s-operators/ci-framework` as it was earlier? 
Or maybe:
```
ci_dcn_site_project_path: ' /home/zuul/src/github.com/openstack-k8s-operators'
ci_dcn_site_arch_repo_path: "{{ cifmw_architecture_repo | default(ci_dcn_site_project_path + '/architecture' }}"
ci_dcn_site_cifmw_repo_path: "{{ ci_dcn_site_project_path + '/ci-framework' }}"
```

Hard to say, what would be easier to debug in the future.